catalog/core/catalog-core-api-impl/src/main/java/ddf/catalog/data/impl/types/CoreAttributes.java

```diff
@@ -171,6 +177,12 @@
                 true /* tokenized */,
                 true /* multivalued */,
                 BasicTypes.STRING_TYPE));
+        descriptors.add(new AttributeDescriptorImpl(DATA_STORE,
+                true /* indexed */,
+                true /* stored */,
+                false /* tokenized */,
+                false /* multivalued */,
```

Comment:
Perhaps it could be, actually, in some use cases I can imagine. It may be better to make it multi-valued just in case.